src/System.Runtime/ref/System.Runtime.cs

```diff
@@ -6139,7 +6139,7 @@ public class TypeDelegator : System.Reflection.TypeInfo
     public delegate bool TypeFilter(Type m, Object filterCriteria);
     public abstract partial class TypeInfo : System.Type, System.Reflection.IReflectableType
     {
-        internal TypeInfo() { }
+        protected TypeInfo() { }
```

Comment:
While considering the future of RefEmit is on our horizon, we're focused on NS2.0 bringup for UWP right now so it's not going to happen in the next few weeks.

If I'm understanding correctly, the best path for preserving our options is to leave the RefEmit contracts the way they are. This work seems to be motivated by completeness rather than real world developers wanting to interoperate TypeInfo's and RefEmit classes. TypeInfo was only created for the rejected .NetCore refactoring and has virtually no use outside of that realm - a realm that doesn't include RefEmit. I'm not seeing evidence of a horde of developers beating down the doors clamoring to cast TypeBuilder to TypeInfo.

@danmosemsft - I'd recommend putting this in the "Consciously decided not to add" bucket.